templates/sync-function/template.js

```diff
@@ -2,7 +2,7 @@
 function synctos(doc, oldDoc) {
   // Whether the given value is either null or undefined
   function isValueNullOrUndefined(value) {
-    return typeof value === 'undefined' || value === null;
+    return value === void 0 || value === null;
```

Comment:
These `void 0` changes aren't directly related to the purpose of this pull request, but this is the preferred approach to test for undefined values.